### PR TITLE
Added kgb card and redirect

### PR DESCRIPTION
**Pull request by Miracle-Aligner**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -10,155 +10,169 @@ var gulpSequence = require('gulp-sequence');
...
 
 gulp.task('templates', function() {
-  return gulp.src('templates/**/*.jade')
-      .pipe(jade({
-        pretty: "\t",
-        locals: JSON.parse(fs.readFileSync("data.json", 'utf8'))
-      }))
-      .pipe(gulp.dest("public"));
-});
+    return gulp.src('templates/**/*.jade')
```
Comment (chudaol) [inline gulpfile.js]: what happened with these spaces? can you check?

File: gulpfile.js [hunk 2]
```diff
@@ -10,155 +10,169 @@ var gulpSequence = require('gulp-sequence');
 var _ = require('underscore');
 var browserSync = require('browser-sync').create();
 
-var books = ["ukrainian", "physics", "maths", "biology", "english", "geography", "history", "law", "english2", "anticorruption-lesson"];
+var books = ["ukrainian", "physics", "maths", "biology", "english", "geography", "history", "law", "english2", "anticorruption-lesson", "vaccination"];
 
 var lessons = ["1", "2"];
```
Comment (chudaol) [inline gulpfile.js]: for some reason command `gulp` doesn't work alone anymore, we need to call `gulp templates` before. Can you check what changed? Ideally `gulp` should do the job

File: gulpfile.js [hunk 3]
```diff
@@ -10,155 +10,169 @@ var gulpSequence = require('gulp-sequence');
...
+      .pipe(gulp.dest('css'));
 });
 
+
 gulp.task('watch', function () {
-  gulp.watch(['js/**/*.js', 'templates/**/*.jade', 'css/*.css', 'css/*.less', 'css/*.scss', 'css/**/*.scss'], ['copy']);
+  gulp.watch('js/**/*.js', 'templates/**/*.jade', 'css/*.css', 'css/*.less', 'css/*.scss', 'css/**/*.scss', gulp.series('copy'));
 });
 
+// gulp.task('default', gulp.series('copy'));
```
Comment (chudaol) [inline gulpfile.js]: remove this line

File: gulpfile.js [hunk 4]
```diff
@@ -10,155 +10,169 @@ var gulpSequence = require('gulp-sequence');
...
+    gulp.src("public/test_vybir_profesii_zhyttia.html")
+        .pipe(rename("index.html"))
+        .pipe(gulp.dest("build/test_vybir_profesii_zhyttia"));
+    gulp.src("public/vaccination.html")
+        .pipe(rename("index.html"))
+        .pipe(gulp.dest("build/vaccination"));
+    gulp.src("public/zno.html")
+        .pipe(rename("index.html"))
+        .pipe(gulp.dest("build/zno"));
+    //gulp.src("public/mun.html")
```
Comment (chudaol) [inline gulpfile.js]: remove commented lines

File: package.json [hunk 5]
```diff
@@ -1,27 +1,34 @@
 {
-  "scripts": {
-    "build": "gulp",
-    "dev": "gulp serve",
-    "start": "http-server ."
-  },
-  "devDependencies": {
+  "name": "landing_page_preproduction",
+  "version": "1.0.0",
```
Comment (chudaol) [inline package.json]: why did you remove `scripts`?

Comment (Miracle-Aligner) [reply package.json]: No special reason except my inattention.

File: gulpfile.js (hunk 4 above)
Comment (Miracle-Aligner) [reply gulpfile.js]: Done

File: gulpfile.js (hunk 2 above)
Comment (Miracle-Aligner) [reply gulpfile.js]: A lot of problems was caused by my poor gulp knowledge.
But there was `[error](https://stackoverflow.com/questions/36897877/gulp-error-the-following-tasks-did-not-complete-did-you-forget-to-signal-async)` running gulpfile that I fixed in last commit.

Comment (chudaol) [reply gulpfile.js]: nice!
